index.html

```diff
@@ -337,30 +337,35 @@ <h2 class="text-secondary-dark">About The Town Hall Project</h2>
           <li><small><a href="mailto:techsupport@townhallproject.com">Report site issues</a></small></li>
         </ul>
       </div>
-    </footer>
+    </footer> 
     <!-- Vendor Files -->
     <script src="/vendor/scripts/jquery-2.1.4.js"></script>
     <script src="/vendor/scripts/jquery-ui.js"></script>
     <script src="/vendor/scripts/bootstrap.min.js"></script>
     <script src="/vendor/scripts/bootstrap3-typeahead.min.js"></script>
     <script src="/vendor/scripts/tether.js"></script>
     <script src="/vendor/scripts/handlebars-v4.0.5.js"></script>
+    <script src="/vendor/scripts/mapbox-gl-v0.32.1.js"></script>
+    <script src="/vendor/scripts/geo-viewport.js"></script>
+    <script src="/data/states.js"></script>
+    <script src="/data/bboxes.js"></script>
+    <script src="/data/zipLookup.js"></script>
+
     <script src="templates.js"></script>
     <script src="https://www.gstatic.com/firebasejs/3.6.6/firebase.js"></script>
 
     <script src='https://actionnetwork.org/widgets/v2/form/stay-up-to-date-on-our-work?format=js&source=widget'></script>
     <script type="text/javascript">(function () {
-            if (window.addtocalendar)if(typeof window.addtocalendar.start == "function")return;
-            if (window.ifaddtocalendar == undefined) { window.ifaddtocalendar = 1;
-                var d = document, s = d.createElement('script'), g = 'getElementsByTagName';
-                s.type = 'text/javascript';s.charset = 'UTF-8';s.async = true;
-                s.src = ('https:' == window.location.protocol ? 'https' : 'http')+'://addtocalendar.com/atc/1.5/atc.min.js';
-                var h = d[g]('body')[0];h.appendChild(s); }})();
+      if (window.addtocalendar)if(typeof window.addtocalendar.start == "function")return;
+      if (window.ifaddtocalendar == undefined) { window.ifaddtocalendar = 1;
+        var d = document, s = d.createElement('script'), g = 'getElementsByTagName';
+        s.type = 'text/javascript';s.charset = 'UTF-8';s.async = true;
+        s.src = ('https:' == window.location.protocol ? 'https' : 'http')+'://addtocalendar.com/atc/1.5/atc.min.js';
+        var h = d[g]('body')[0];h.appendChild(s); }})();
     </script>
     <!-- Script Files -->
     <script src="/scripts/models/event.js"></script>
     <script src="/scripts/views/mapView.js"></script>
-    <script src="/scripts/views/eventView.js"></script>
```

Comment:
We still need eventView for functionality like rendering the MoC cards and setting up the typeaheads etc.